packages/babel-core/src/config/full.js

```diff
@@ -45,10 +47,10 @@ type SimpleContext = {
   caller: CallerMetadata | void,
 };
 
-export default function loadFullConfig(
+export default gensync<[any], ResolvedConfig | null>(function* loadFullConfig(
```

Comment:
Why the `any` instead of `mixed`?

Reply:
It's because I get errors about objects not being compatible with `mixed`:
https://github.com/babel/babel/blob/0da4f7ef50b5299d67d7bd3a10fb66453f214dc3/packages/babel-core/src/transformation/block-hoist-plugin.js#L14-L18
https://github.com/babel/babel/blob/0da4f7ef50b5299d67d7bd3a10fb66453f214dc3/packages/babel-core/src/config/index.js#L18-L19

Reply:
Huh, seems like a Flow bug, but fair enough.